.github/workflows/skywalking.yaml

```diff
@@ -560,6 +561,13 @@ jobs:
               ISTIO_VERSION=1.18.0
               KUBERNETES_VERSION=25
 
+          # TODO, disable Istio relative e2e tests temporarily,
+          # as they are blocked by the GHA resource limitation.
+          # - name: Rover with Istio Process 1.15.0
+          #   config: test/e2e-v2/cases/rover/process/istio/e2e.yaml
+          #   env: ISTIO_VERSION=1.15.0
+          #   runs-on: ubuntu-20.04
+
```

Comment:
@kezhenxu94 This seems to have to be disabled in this way. The final check is for the whole matrix.
I think no matter what this is, we could revert this PR once your fix is done.